<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibrarySearchDockWidget</class>
 <widget class="QDockWidget" name="LibrarySearchDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library Search</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" colspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="searchPathTab">
       <attribute name="title">
        <string>Sources</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0" colspan="3">
         <layout class="QGridLayout" name="fileSystemLayout"/>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QPushButton" name="recompileButton">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Recompile</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="checkAllButton_2">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>all</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="checkNoneButton_3">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>none</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="queryDiagramTab">
       <attribute name="title">
        <string>Query</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="searchResultsTab">
       <attribute name="title">
        <string>Results</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1" colspan="2">
         <widget class="QComboBox" name="orderByComboBox">
          <item>
           <property name="text">
            <string>alphabetical on name</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Order by:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <layout class="QVBoxLayout" name="searchResultsLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QPushButton" name="searchButton">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Search for what is selected in the current diagram view (see tabs).</string>
      </property>
      <property name="text">
       <string>üîç Search library</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
